project/dexter-monitor/public/ctrl/overviewCtrl.js

```diff
@@ -0,0 +1,151 @@
+        {field:'defectCountTotal',      displayName:'Total',            width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Number of all defects',         aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'defectCountFixed',      displayName:'Fixed',            width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Number of fixed defects',       aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'defectCountDismissed',  displayName:'Dismissed',        width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Number of dismissed defects',   aggregationType: uiGridConstants.aggregationTypes.sum}
+    ];
+
+    initialize();
+
+    function initialize() {
+        $scope.summaryGridOptions = createGrid(summaryColumnDefs);
+        removeUselessGridOptions($scope.summaryGridOptions);
+        removeScrollbarFromGrid($scope.summaryGridOptions);
+        $scope.installationStatusGridOptions = createGrid(installationStatusColumnDefs);
+        $scope.installationStatusGridOptions.showColumnFooter = true;
+        removeScrollbarFromGrid($scope.installationStatusGridOptions);
+        $scope.defectStatusGridOptions = createGrid(defectStatusColumnDefs);
+        $scope.defectStatusGridOptions.showColumnFooter = true;
+        removeScrollbarFromGrid($scope.defectStatusGridOptions);
+
+        $scope.time = new Date().toLocaleString();
+        $scope.summaryGridOptions.data.push({});
+        loadDataForInstallationStatusGrid();
+        loadDataForDefectStatusGrid();
+        setGridExportingFileNames($scope.installationStatusGridOptions, INSTALLATION_STATUS_FILENAME_PREFIX + '-' + $scope.time);
+        setGridExportingFileNames($scope.defectStatusGridOptions, DEFECT_STATUS_FILENAME_PREFIX + '-' + $scope.time);
+    }
+
+    function removeScrollbarFromGrid(gridOptions) {
+        gridOptions.enableHorizontalScrollbar = uiGridConstants.scrollbars.NEVER;
+        gridOptions.enableVerticalScrollbar =  uiGridConstants.scrollbars.NEVER;
+    }
+
+    function removeUselessGridOptions(gridOptions) {
+        gridOptions.enableSorting = false;
+        gridOptions.enableFiltering = false;
+        gridOptions.showGridFooter = false;
+        gridOptions.enableGridMenu = false;
+    }
+
+    function loadDataForInstallationStatusGrid() {
+        UserService.getUserStatus()
+            .then((rows) => {
+                $scope.installationStatusGridOptions.data = rows;
+                resizeHeightOfGrid('overviewInstallationStatusGrid', rows.length);
+                const targetDeveloperCountTotal = _.sum(_.map($scope.installationStatusGridOptions.data, 'targetDeveloperCount'));
+                const installedDeveloperCountTotal = _.sum(_.map($scope.installationStatusGridOptions.data, 'installedDeveloperCount'));
+                $scope.rate = ((installedDeveloperCountTotal / targetDeveloperCountTotal) * 100).toFixed(1);
```

Comment:
Please check whether 'targetDevelopercountTotal is zero or not, before using it.